vagrant/boxes.d/99-local.yaml.example

```diff
@@ -96,3 +96,13 @@ centos7-squid:
       options:
         dev: 'virbr3'
         type: 'bridge'
+
+centos7-foreman-smoker:
```

Comment:
Rather than a separate box, does it make sense to include it by default in the devel box? Technically there's no need for a separate box other than guaranteeing that there's separation between services. We capture that in our pipelines and can cover it in reviews.

Reply:
I would rather keep a separate box for a few reasons:
1) If the steps in these roles are failing for any reason that will mean the devel box will fail to spin up
2) Being able to spin up different boxes and run smoker tests on them locally. I could see that being helpful for multiple development environments, foreman/katello/luna environments, and releases.
3) We can stay flexible as we figure out what the expectations are of the environment(s) that smoker will run on.